src/main/java/lotto/step2/enums/MatchNumber.java

```diff
@@ -0,0 +1,61 @@
+package lotto.step2.enums;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public enum MatchNumber {
+    FOURTH(3, 5000, "3개 일치 (5,000원)- "),
+    THIRD(4, 50000, "4개 일치 (50,000원)- "),
+    SECOND(5, 1500000, "5개 일치 (1,500,000원)- "),
+    FIRST(6, 200000000, "6개 일치 (2,000,000,000억)- ");
```

Comment:
자바에선 이렇게 큰 숫자에 대해서 다음과 같이 언더바를 통해 가독성을 높힐 수 있습니다.
```suggestion
    FIRST(6, 200_000_000, "6개 일치 (2,000,000,000억)- ");
```